jenkins-remoting.yaml

```
@@ -0,0 +1,59 @@
+package:
+  name: jenkins-remoting
+  version: 3206.vb_15dcf73f6a_9
```

we don't support this version format might need something like this 
```suggestion
  version: 3206
```

---

Cause Jenkins versioning https://github.com/jenkinsci/jep/blob/master/jep/305/README.adoc,"3206" number of commits reachable from the HEAD of the current branch (git rev-list --first-parent --count HEAD) ,could we keep full version name ? if only "3206" I think that quite difficult to tracking version

---

we can use 3206_git[date] where [date] is the date of the snapshot in YYYYMMDD format.